### ISSUE TITLE
Main.java Error

### ISSUE THREAD

**Issue by zzhangyue** [nus-cs2030/2223-s2#541]
# Summary

Hiii, would appreciate some help on this...

## Description:

I am not quite sure what am I missing...oop

![Screenshot 2023-03-23 at 10 46 07 (2)](https://user-images.githubusercontent.com/122248591/227087310-4a48ce0e-b46b-4307-8921-148e3ecc1289.png)
<img width="454" alt="Screenshot 2023-03-23 at 10 45 20" src="https://user-images.githubusercontent.com/122248591/227087337-e7b64bab-3aa8-4471-9280-436dbc55a477.png">

**Comment by sethteo** [comment]
You can remove the word "public" from line 3 and submit it as class Main {
~~~
}
Should work afterwards :)

**Comment by ayevxn** [comment]
Yes I faced this same issue, fixed it by removing 'public' before class Main.

**Comment by dlyxxx** [comment]
I didnt have the public in there either!

**Comment by ryantangmj** [comment]
Removing the public in front of the main will fix this

**Comment by mavisczx** [comment]
hello, for me I did not remove the public but added this comment in line 4:
/**
     * Print the parameter needed.
     * @param args The parameter to be printed 
     * 
     */

**Comment by e0968868** [comment]
If you remove "public", it should work fine!

**Comment by NwaterM** [comment]
i faced the same problem! just need remove public !
